lib/utils.js

```diff
@@ -571,15 +571,27 @@ function deleteLogProperty (log, property) {
 }
 
 /**
- * Filter a log object by removing any ignored keys.
+ * Filter a log object by removing or including keys accordingly.
+ * When `includeKeys` is passed, `ignoredKeys` will be ignored.
  *
  * @param {object} log The log object to be modified.
  * @param {Set<string> | Array<string>} ignoreKeys An array of strings identifying the properties to be removed.
+ * @param {Set<string> | Array<string>} includeKeys An array of strings identifying the properties to be included.
  *
  * @returns {object} A new `log` object instance that does not include the ignored keys.
  */
-function filterLog (log, ignoreKeys) {
+function filterLog (log, ignoreKeys, includeKeys) {
```

Comment:
I'd rather change this signature. Clearly we are going to keep adding to it.

```suggestion
function filterLog ({ log, ignoreKeys = [], includeKeys = [] } = {}) {
```

Reply:
Also updated the `help.txt` in 2bf5c72.